README.md

```diff
@@ -57,37 +55,25 @@ $ npm install --save sumo-logger
 **From GitHub:**
 * Download or clone this repo.
 * Copy the files in the `src` folder into your app/website source.
-* Add `<script src="path/to/sumologic.logger.min.js"></source>` or
-   `<script src="path/to/sumologic.logger.js"></source>` to your pages.
-* Add a `<script>` block with the desired log call, as explained in [Usage](#user-content-usage), to your pages.
+* Add `<script src="path/to/sumologic.logger.js"></source>` to your pages or `import` or `require` the module in your app.
```

Comment:
This appears to still encourage use of the old `sumologic.logger.js` library, is it possible to remove?

Reply:
good catch